README.md

```diff
@@ -226,7 +249,7 @@ module "mod_vnet_hub" {
 
 Hub Networking is set up in a Management Hub Overlay design based on the SCCA Hub/Spoke architecture. The Management Hub Overlay is a central point of connectivity to many different networks.
 
-The following parameters affect Management Hub Overlay Networking.
+The following parameters affect Management Virtual Network Hub Overlaying.
```

Comment:
Typo?  "Should 'Virtual Network Hub Overlaying' be "Virtual Network Hub Overlay'?

Reply:
ADDED